MdeModulePkg/MdeModulePkg.dec

```diff
@@ -2049,7 +2049,7 @@
   #   TRUE  - Create boot options for pre-installed OSes.<BR>
   #   FALSE - Do not create boot options for pre-installed OSes.
   # @Prompt Create boot options for pre-installed OSes.
-  gEfiMdeModulePkgTokenSpaceGuid.PcdCreatePreInstalledBootOptions|FALSE|BOOLEAN|0x0001007a
+  gEfiMdeModulePkgTokenSpaceGuid.PcdCreatePreInstalledBootOptions|TRUE|BOOLEAN|0x0001007a

```

Comment:
@filipleple We do not want to change the default value in this module.
We need to override this value just for OvmfPkg.